src/executables/util_cs.c

```diff
@@ -4000,6 +4000,81 @@ tde (UTIL_FUNCTION_ARG * arg)
   return EXIT_FAILURE;
 }
 
+static int
+check_table_name (const char *table_name)
+{
+  int table_name_len = STATIC_CAST (int, strlen (table_name));
+  int sub_len = 0;
+  const char *dot = NULL;
+
+  if (table_name_len >= SM_MAX_IDENTIFIER_LENGTH)
+    {
+      PRINT_AND_LOG_ERR_MSG (msgcat_message
+			     (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_FLASHBACK,
+			      FLASHBACK_MSG_EXCEED_MAX_CLASSNAME_LENGTH), SM_MAX_USER_LENGTH,
```

Comment:
When I tested, I was able to create a user with a length of 31 instead of 32.

Reply:
Thanks for testing.

Yes, that is intended result.

The length of the owner_name.class_name in 'user schema' is described as follows.
> Length = user_name (31 bytes) + dot (1 byte) + object_name (222 bytes) + null (\0) (1 byte)

So, I asked the user to set the owner name to less than 32 bytes through an error message.